src/test/java/com/xceptance/neodymium/testclasses/browser/DisableRandomBrowserAnnotation.java

```diff
@@ -0,0 +1,21 @@
+package com.xceptance.neodymium.testclasses.browser;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.RandomBrowsers;
+
+@RunWith(NeodymiumRunner.class)
+@Browser("Chrome_1024x768")
+@Browser("Chrome_1500x1000")
+public class DisableRandomBrowserAnnotation
```

Comment:
please add @RandomBrowsers(1) and verify the disabling

Reply:
Done